kml/types_v8.hpp

```diff
@@ -46,6 +46,33 @@ struct BookmarkDataV8
 
   bool operator!=(BookmarkDataV8 const & data) const { return !operator==(data); }
 
+  BookmarkDataV8()
+  {
+
+  }
```

Comment:
Is it really needed?
```suggestion
  BookmarkDataV8() = default;
```

Reply:
Fixed.
We need default constructor to initialize `std::vector<BookmarkDataV8> m_bookmarksData;` field in FileDataV8